lib/src/react_client/event_helpers.dart

```diff
@@ -0,0 +1,324 @@
+// ignore_for_file: deprecated_member_use_from_same_package
+
+import '../../react.dart';
+
+/// Returns an empty [SyntheticEvent] with all fields set to `null`.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticEvent getEmptySyntheticEvent() => buildSyntheticEvent();
+SyntheticClipboardEvent getEmptyClipboardEvent() => buildClipboardEvent();
+SyntheticKeyboardEvent getEmptyKeyboardEvent() => buildKeyboardEvent();
+SyntheticCompositionEvent getEmptyCompositionEvent() => buildCompositionEvent();
+SyntheticFocusEvent getEmptyFocusEvent() => buildFocusEvent();
+SyntheticFormEvent getEmptyFormEvent() => buildFormEvent();
+SyntheticMouseEvent getEmptyMouseEvent() => buildMouseEvent();
+SyntheticPointerEvent getEmptyPointerEvent() => buildPointerEvent();
+SyntheticTouchEvent getEmptyTouchEvent() => buildTouchEvent();
+SyntheticAnimationEvent getEmptyAnimationEvent() => buildAnimationEvent();
+SyntheticUIEvent getEmptyUIEvent() => buildUIEvent();
+SyntheticWheelEvent getEmptyWheelEvent() => buildWheelEvent();
+
+/// Returns a newly constructed [SyntheticEvent] instance.
+///
+/// In addition to creating empty instances (by invoking without any parameters) or completely custom instances
+/// by using [properties], this method will merge previously existing [baseEvent]s with the [properties] provided.
+/// [properties] takes precedence, and therefore can be used to override specific fields on the [baseEvent].
+SyntheticEvent buildSyntheticEvent({SyntheticEvent baseEvent, Map<String, dynamic> properties = const {}}) {
```

Comment:
#nit I think "create" would be better in this context; "build" might imply there's a builder object involved."

So, I'd prefer `createSyntheticEvent` instead of `buildSyntheticEvent`.